quantmsio/core/sdrf.py

```
@@ -124,7 +124,9 @@ def _load_sdrf_info(self, sdrf_file: Union[Path, str]):
         try:
             self.sdrf_table = pd.read_csv(sdrf_file, sep="\t", header=0)
         except FileNotFoundError:
-            raise FileNotFoundError("The SDRF file provided not found: " + sdrf_file)
+            raise FileNotFoundError(
+                "The SDRF file provided not found: " + str(sdrf_file)
+            )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)